tslint.json

```diff
@@ -49,7 +49,8 @@
       true,
       "ban-keywords",
       "check-format",
-      "allow-leading-underscore"
+      "allow-leading-underscore",
+      "allow-pascal-case"
```

Comment:
I had to allow `allow-pascal-case` in order to use captial start letter when writing styled-components.